Src/BootCamp.Chapter/Computer/ComputerCase.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter.Computer
+{
+    public class ComputerCase
+    {
+        private string _manufacturersId;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make '_manufacturersId' 'readonly'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5121609)